src/main/java/model/process/FromProcess.java

```diff
@@ -11,14 +17,29 @@
  */
 public class FromProcess extends DataProcess {
 
-	private Identifier<DataTable> identifier;
+	private Identifier<DataTable>[] identifiers;
 
-	public FromProcess(Identifier<DataTable> identifier) {
-		this.identifier = identifier;
+	public FromProcess(Identifier<DataTable>... identifiers) {
+		this.identifiers = identifiers;
 	}
 
 	@Override
 	protected Table doProcess() {
-		return getDataModel().getByName(identifier.getName()).copy();
+		Supplier<Stream<DataTable>> tableStream = () ->
+				Arrays.stream(identifiers)
+				.map(x -> getDataModel().getByName(x.getName()).copy());
+		Optional<DataTable> first = tableStream.get().findFirst();
+		DataTable[] tables = tableStream.get().skip(1).toArray(DataTable[]::new);
+
+		if (!first.isPresent()) {
+			throw new IllegalArgumentException(
```

Comment:
Ik heb er nu ook een test voor toegevoegd, zodat hij waarschijnlijk zal falen als setCodes gemerged is.